### ISSUE TITLE
envsubst doesn't work

### ISSUE THREAD

**Issue by btmorex** [RanadeepPolavarapu/docker-nginx-http3#7]
I get the following when trying to use envsubst with a container based on your image:

```
proxy_1   | Error loading shared library libintl.so.8: No such file or directory (needed by /usr/local/bin/envsubst)
proxy_1   | Error relocating /usr/local/bin/envsubst: libintl_bindtextdomain: symbol not found
proxy_1   | Error relocating /usr/local/bin/envsubst: libintl_gettext: symbol not found
proxy_1   | Error relocating /usr/local/bin/envsubst: libintl_textdomain: symbol not found
```

**Comment by RanadeepPolavarapu** [comment]
Hi,

IIRC `envsubst` needs to be explicitly installed. 

You can use this image as the base and add the package:

```Dockerfile
# Base Nginx HTTP/3 Image
FROM ranadeeppolavarapu/nginx-http3:latest

RUN apk add gettext libintl \
    && mv /usr/bin/envsubst /usr/local/sbin/envsubst
```

This should work.

**Comment by btmorex** [comment]
Yeah, libintl is required, which I added to my build. I guess my point was there's no point in add envsubst to your image (it is there in /usr/local/bin) if you don't also add libintl because it simply won't work.

**Comment by RanadeepPolavarapu** [comment]
Resolved in c3337a315e4306921b22350e85aa7f6b52902c66.
